dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapperTest.java

```diff
@@ -746,6 +739,7 @@ void shouldMapRequestedOrgUnitAsSelectedWhenOrgUnitProvidedAndNoOrgUnitModeProvi
         EventOperationParams.builder()
             .programUid(program.getUid())
             .orgUnitUid(orgUnit.getUid())
+            .orgUnitMode(SELECTED)
```

Comment:
The test name does not make sense I think

`shouldMapRequestedOrgUnitAsSelectedWhenOrgUnitProvidedAndNoOrgUnitModeProvided`

you do provide a mode now

Reply:
Right, it's another test that has no value after the latest commits